src/Main.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import './Main.css';
+import NewTaskForm from './NewTaskForm';
+import AddedTasks from './AddedTasks';
+import EditTaskForm from './EditTaskForm';
+
+export default class Main extends Component {
+  state = {
+    isOpenNewTaskForm: false,
+    isOpenEditTaskForm: false,
+    tasks: [],
+    dataEditTask: null
+  };
+
+  render() {
+    const newTaskForm = this.state.isOpenNewTaskForm && <NewTaskForm 
```

Comment:
Think how to merge NewTaskForm and EditTaskForm in one component and use it for all both cases.